ludwig/models/predictor.py

```diff
@@ -317,7 +317,8 @@ def save_prediction_outputs(
     backend,
 ):
     postprocessed_output, column_shapes = flatten_df(postprocessed_output, backend)
-    postprocessed_output.to_parquet(os.path.join(output_directory, PREDICTIONS_PARQUET_FILE_NAME), schema=None)
+    schema = get_pa_schema(postprocessed_output)
```

Comment:
Do we need to manually pass in the schema to `to_parquet()` here given that we already make this call within `to_parquet()` here? https://github.com/ludwig-ai/ludwig/pull/2550/files#diff-fec65d5ff8d68e96ebf11e5114a25e0ffa187739296f1e89d9949ae04cef0cc1R188-R195

Reply:
Good callout– didn't realize that the `backend` variable was being passed in. Used that instead. Thanks!